Index: software-properties-0.60.debian/data/designer/main.ui
===================================================================
--- software-properties-0.60.debian.orig/data/designer/main.ui	2010-10-26 00:18:11.651264047 +0200
+++ software-properties-0.60.debian/data/designer/main.ui	2010-10-26 00:18:22.766761906 +0200
@@ -26,7 +26,7 @@
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
-       <string>Kubuntu Software</string>
+       <string>Debian Software</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
